### ISSUE TITLE
single player forge crash latest build

### ISSUE THREAD

**Issue by mov51** [TVPT/VoxelGunsmith#12]
i can load forge with the VoxelSniper-7.1.0-SNAPSHOT-all.jar and it shows up in mods along with the only other mod (other than the recommended 1.8 forge) world edit (also 1.8). as soon as i load a world it  kicks me to the start screen and when i try again the whole game crashes.

here is the crash it's rather short but i can't make any sense of it.
http://pastebin.com/6TTt4uZG

**Comment by mov51** [comment]
please help i have tried previous build up until it won't even load as a mod none of them work :(

**Comment by TheGurw** [comment]
Similar issue, same build, also SP. Running Forge and Liteloader if that makes any difference, can include version numbers if needed.

Stacktrace starting from first error all the way to end: http://pastebin.com/8HdmCYs7

**Comment by mov51** [comment]
is anyone going to let us know if the issue is being fixed or looked into or anything please?

**Comment by gabizou** [comment]
Yes, @Deamon5550 will be fixing this soon, there's some interactions going on for checking whether VoxelSniper is being loaded by Forge only versus Forge with Sponge, in which case the Forge related code is disabled and VoxelSniper is switched over to Sponge. He's been a bit busy over the last few days, but he is aware of the issue and is working on a fix.

**Comment by mov51** [comment]
ok thank you i was just getting frustrated because i posted this over two week ago and only got another person with the same problem. thank you for letting us know.

**Comment by Deamon5550** [comment]
Its an issue with reobfuscation and the shadowJar not reobfuscating I made an attempt a fixing it earlier but no luck. And frankly the forge version just isn't a priority, if you want to use sniper and forge together just add sponge to your server/client and it'll load the sponge version.

**Comment by mov51** [comment]
there is a mod that add permission (forge essentials) i don't know if that will help or not. maybe you could add a dependency?

what do you mean add sponge to my client i thought sponge was a sever host software

**Comment by Deamon5550** [comment]
SpongeForge at least is a forge mod which will run on the server or client.

**Comment by mov51** [comment]
thanks for letting me know i will look in to it in the morning

**Comment by mov51** [comment]
that half worked. it loaded the world. said i required permissions to use voxelsniper commands then when i quit the world i got an "exception in server tick loop" crash.
